das_client/lib/app/pages/journey/train_journey/widgets/header/extended_menu.dart

```diff
@@ -0,0 +1,239 @@
+import 'package:das_client/app/bloc/train_journey_cubit.dart';
+import 'package:das_client/app/i18n/i18n.dart';
+import 'package:das_client/app/pages/journey/train_journey/widgets/table/config/train_journey_settings.dart';
+import 'package:das_client/app/widgets/assets.dart';
+import 'package:das_client/app/widgets/das_text_styles.dart';
+import 'package:das_client/di.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:sbb_design_system_mobile/sbb_design_system_mobile.dart';
+
+class ExtendedMenu extends StatefulWidget {
```

Comment:
Ich würde den generellen Code für das Popup Menü in den `Widgets` Ordner auslagern, damit es wiederverwendet werden kann.